scripts/spell-caster/src/executeSpell.ts

```diff
@@ -0,0 +1,37 @@
+import assert from 'node:assert'
+import { Address, encodeFunctionData } from 'viem'
+import { getViemClient } from './blockchain/ViemClient'
+import { ChainConfig } from './config'
+
+export async function executeSpell({
+  spellAddress,
+  chain,
+  client,
+}: {
+  spellAddress: Address
+  chain: ChainConfig
+  client: ReturnType<typeof getViemClient>
+}) {
+  const code = await client.getCode({ address: spellAddress })
+  assert(code, `Spell not deployed (address=${spellAddress})`)
+  await client.setCode({
+    address: chain.sparkSpellExecutor,
+    bytecode: code,
+  })
+
+  await client.sendTransaction({
```

Comment:
Can we instead "prank" as the Maker pause proxy address and call execute on the executor with the address of the spell directly? Would be the same as mainnet this way

Reply:
Also would keep the executor code the same on the chain

Reply:
I thought about this. The problem is that doing it like you described doesn't work well on L2s and require more ceremony there (mocking cross chain calls). 

The current approach works well on every domain but I agree that it's not without cons.

Reply:
As discussed, tweaked it to revert to old executor code.